### DISCUSSION TITLE
help with Email templates implementation

### DISCUSSION THREAD

**Discussion by jay-kanakala** [ory/kratos#1374] [Q&A] [answered]
Hi guys,

I am following the quick start tutorial exactly running the app with docker compose. I saw the documentation for overriding email templates path like this

template_override_path: /conf/courier-templates

but when I open the code folder, I cant find this /conf folder.... where is that folder /conf ?
should I create it?

I tried like this

template_override_path: /contrib/quickstart/kratos/courier-templates/
and pasted the templates folder in it (I copied it from /courier/templates)

but it says - template_override_path: /contrib/quickstart/kratos/courier-templates/recovery/valid/email.body.gotmpl : no such file or directory

Any help pls...

Thank you

**Comment by jay-kanakala** [comment]
I even tried to edit the default templates in /courier/templates. But it is not working. I added an extra line in recovery valid template. and tested it, but the extra line is not showing in the mail received.

**Reply by vinckr** [reply]
Sorry for the late response @jay-kanakala, not sure what the problem is. 
Is your kratos.yml loaded correctly where you added the line 
`template_override_path: /contrib/quickstart/kratos/courier-templates/` ?
Best would be if you give me a repository to reproduce that would make checking it out easiest 🙏

**Comment by zepatrik** [comment, marked as answer]
I guess you are confusing your local filesystem with that of the docker container. Locally, you have to create some directory containing all templates. Then you mount that directory as a volume: https://docs.docker.com/storage/volumes/ and https://docs.docker.com/compose/compose-file/compose-file-v3/#volumes

**Comment by chengyu-luna** [comment]
kratos.yml: template_override_path: /conf/courier-templates

quickstart.yml:
kratos:
  volumes:
    - type: bind
        source: ./contrib/quickstart/kratos/courier-templates
        target: /conf/courier-templates
